lib/Service/ImageService.php

```diff
@@ -394,6 +452,26 @@ private function getOrCreateAttachmentDirectoryForFile(File $textFile): ?Folder
 		return null;
 	}
 
+	/**
+	 * Get a user file from file ID
+	 *
+	 * @param int $filePath
+	 * @param string $userId
+	 * @return Folder|null
+	 * @throws \OCP\Files\NotPermittedException
+	 * @throws \OC\User\NoUserException
+	 */
+	private function getFileFromPath(string $filePath, string $userId): ?File {
+		$userFolder = $this->rootFolder->getUserFolder($userId);
+		if ($userFolder->nodeExists($filePath)) {
+			$file = $userFolder->get($filePath);
```

Comment:
Will throw a NotFoundException so would be good to also annotate that if not catches